cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/model/Replica.java

```diff
@@ -115,34 +115,36 @@ void clearLoad() {
   /**
    * (1) Remove leadership from the replica.
    * (2) Clear and get the outbound network load associated with leadership from the given replica.
+   * (3) Clear and get the CPU leadership load associated with leadership from the given replica.
    *
-   * @return Removed leadership load by snapshot time -- i.e. outbound network load by snapshot time.
+   * @return Removed leadership load by snapshot time -- i.e. outbound network and fraction of CPU load by snapshot time.
    */
   Map<Resource, Map<Long, Double>> makeFollower()
       throws ModelInputException {
     // Remove leadership from the replica.
     setLeadership(false);
-    // Clear and get the outbound network load associated with leadership from the given replica.
+    // Clear and get the outbound network and CPU load associated with leadership from the given replica.
     Map<Long, Double> leadershipNwOutLoad = _load.loadFor(Resource.NW_OUT);
-    Map<Long, Double> leadershipCpuLoad = _load.loadFor(Resource.CPU);
+    Map<Long, Double> cpuLoadOfLeader = _load.loadFor(Resource.CPU);
```

Comment:
For consistency, "leaderCpuLoad" seems better.

Reply:
Fixed.